CHANGELOG.md

```diff
@@ -34,13 +43,24 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - [#454](https://github.com/JabRef/jabref/issues/454) Add a tab that shows all remaining entry fields that are not displayed in any other tab
 - The LaTeX to Unicode/HTML functionality is much improved by covering many more cases
 - Ability to convert from LaTeX to Unicode in right-click field menu
+- Regex-based search is know only applied entirely and not split up to different Regexes on whitespaces
+- Application now starts from the JavaFX thread and the about dialog is now implemented in JavaFX
```

Comment:
Since the Changelog should be understandable for the average user, I would propose something like `Partly switched to new UI technology (JavaFX). Redesigned about dialog.̀